PineapplePlanner/PineapplePlanner.Infrastructure/FirestoreService.cs

```diff
@@ -16,7 +16,9 @@ public FirestoreDb FirestoreDb
 
         public FirestoreService()
         {
-            GoogleCredential credential = GoogleCredential.FromFile("firebase-adminsdk-key.json");
+            // Credential file should be stored in %APPDATA%/PineapplePlanner/firebase-key.json
+            String credentialPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "PineapplePlanner\\firebase-key.json");
+            GoogleCredential credential = GoogleCredential.FromFile(credentialPath);
```

Comment:
Please stick to the original approach. If you want to change the project configuration, please explain why and adapt the documentation in the README.md.